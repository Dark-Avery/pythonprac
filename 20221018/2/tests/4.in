:sin x y z w sin(x+y+z+w)
sin 1 -2 2 -1
:decorate s g h j "--<<{},{},{},{}>>--".format(s,g,h,j)
decorate "ЖЖЖ" "УУУ" "ЖЖЖ" "УУУ"
sin 0 1 -1 1
quit "все будет хорошо!! {}, {}"