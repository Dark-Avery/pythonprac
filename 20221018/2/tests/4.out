0.0
--<<ЖЖЖ,УУУ,ЖЖЖ,УУУ>>--
0.8414709848078965
все будет хорошо!! 3, 6
